dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/interruptedjobwindow/JobInfoPresenter.java

```diff
@@ -89,20 +76,25 @@ private void initListeners() {
         cancelJobListButtons.setOnAction(event -> {
             if (CheckNetwork.isReachable(endpointInfo.getClient())) {
                 final Map<String, FilesAndFolderMap> jobIDMap = ParseJobInterruptionMap.getJobIDMap(jobInterruptionStore.getJobIdsModel().getEndpoints(), endpointInfo.getEndpoint(), endpointInfo.getDeepStorageBrowserPresenter().getJobProgressView(), null);
-                if (jobIDMap != null && jobIDMap.size() != 0) {
-                    final Button exitButton = (Button) CLOSECONFIRMATIONALERT.getDialogPane().lookupButton(
+                if (!Guard.isMapNullOrEmpty(jobIDMap)) {
+
+                    final Alert closeconfirmationalert = new Alert(
+                            Alert.AlertType.CONFIRMATION,
+                            ""
```

Comment:
I was just copying what was already there.  I'm not sure if there should be any text or not.